src/clj/rems/home.clj

```diff
@@ -36,17 +38,32 @@
   easy memoization purposes."
   [language]
   (log/info (str "Rendering stylesheet for language " language))
-  (-> (styles/screen)
+  (-> (styles/screen-css)
       (response)
       (content-type "text/css")))
 
 (def memoized-render-css (memoize render-css))
 
-(defroutes home-routes
+(defroutes normal-routes
   (GET "/" [] (layout/home-page))
   (GET "/accept-invitation" {{:keys [token]} :params} (redirect (str "/#/application/accept-invitation/" token)))
   (GET "/apply-for" {{:keys [resource]} :params} (apply-for-resource resource))
   (GET "/landing_page" req (redirect "/#/redirect")) ; DEPRECATED: legacy url redirect
   (GET "/markdown/:filename" [filename] (markdown-page filename))
-  (GET "/favicon.ico" [] (redirect "/img/favicon.ico"))
-  (GET "/css/screen.css" [] (memoized-render-css context/*lang*)))
+  (GET "/favicon.ico" [] (redirect "/img/favicon.ico")))
+
+(defn- get-language-from-filename [filename]
+  (keyword (subs filename (inc (str/index-of filename "-")) (str/index-of filename "."))))
```

Comment:
just a minor thing but I'd prefer a regex to two subs+index-of+index-of

Reply:
I don't for this simple case but fortunately this piece of code is not needed anymore.